sdks/python/apache_beam/runners/portability/stager.py

```diff
@@ -476,25 +466,71 @@ def _is_remote_path(path):
     return path.find('://') != -1
 
   @staticmethod
-  def _create_jar_packages(jar_packages, temp_dir):
+  def append_jar_packages_to_java_env(proto_pipeline, options, temp_dir=None):
+    """Append a list of jar packages to Docker Java SDK Environment.
```

Comment:
So jar_packages is still needed ? Is this only for testing ?
In production, users will get Java dependencies from CLASSPATH of the expansion service, right ? I wonder if we can migrate existing tests to that path as well and remove this code.